extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -244,14 +244,16 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
   }
 
   /** @override */
-  getAdUrl(consentState) {
+  getAdUrl(consentTuple) {
     if (
-      consentState == CONSENT_POLICY_STATE.UNKNOWN &&
+      !consentTuple ||
```

Comment:
As mentioned previously, do not see how this would ever be nullable and so you can remove this check.  Also in general make sure to wrap portions of predicate in parens when mixing or and and operators